tests/integrations/python/tests/test_openai.py

```
@@ -2832,3 +2834,415 @@ def test_51c_input_tokens_long_text(self, provider, model, vk_enabled):
+        if len(annotations_found) > 0:
+            print(f"✓ Found {len(annotations_found)} annotations")
+            for i, annotation in enumerate(annotations_found[:3]):
+                assert_valid_openai_annotation(annotation, expected_type="url_citation")
+                print(f"  Annotation {i+1}:")
+                print(f"    Type: {annotation.type}")
+                print(f"    URL: {annotation.url if hasattr(annotation, 'url') else 'N/A'}")
+                if hasattr(annotation, "title"):
+                    print(f"    Title: {annotation.title}")
+                # Check for encrypted_index preservation
+                if hasattr(annotation, "encrypted_index"):
+                    print(f"    Encrypted index present: ✓")
```

Ruff (0.14.11)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)